networking/enable-cluster-wide-proxy.adoc

```diff
@@ -7,7 +7,19 @@ toc::[]
 
 Production environments can deny direct access to the Internet and instead have an HTTP or HTTPS proxy available. You can configure {product-title} to use a proxy by xref:../networking/enable-cluster-wide-proxy.adoc#nw-proxy-configure-object_config-cluster-wide-proxy[modifying the Proxy object for existing clusters] or by configuring the proxy settings in the `install-config.yaml` file for new clusters.
 
-IMPORTANT: The cluster-wide proxy is only supported if you used a user-provisioned infrastructure installation for a supported provider.
+[IMPORTANT]
+====
+The cluster-wide proxy is only supported if you used a user-provisioned infrastructure installation for a supported provider.
+====
+
+.Prerequisites
+
+* Review the  xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any need to bypass the proxy. By default, all cluster egress traffic is proxied, including calls to hosting cloud provider APIs. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
```

Comment:
calls to instance metadata endpoint is not proxied: https://github.com/openshift/cluster-network-operator/blob/master/pkg/util/proxyconfig/no_proxy.go#L65-L68

The metadata ip (and dns name for gcp) is automatically added to no proxy by the cluster network operator.